eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,19 +1,26 @@
 parameters:
   Directory: 'not-specified'
   IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  
+  WorkingDirectory: "$(System.DefaultWorkingDirectory)"
+  ScriptDirectory: "eng/common/scripts"
+  Urls: "$(dir -r -i *.md)"
```

Comment:
1. With powershell scripts, it's better not to use aliases in automation. Use `get-childitem`. Aliases could be overridden.
2. Being brief with arguments could actually create conflicts in the future if another argument is added that starts wit the same letter. Better to be explicit, i.e. `-recurse` and `-include`, however...
3. Use `-filter` instead of `-include`. `-filter *.md` actually uses file system support to filter and is much faster than `-include` that filters after enumeration (and is supported by all namespace providers, while `-filter` works with the file system provide and maybe one or two others).

Reply:
Fixed the alias commands